src/proto/h1/conn.rs

```diff
@@ -649,6 +655,7 @@ impl fmt::Debug for State {
             .field("keep_alive", &self.keep_alive)
             .field("error", &self.error)
             //.field("method", &self.method)
+            .field("title_case_headers", &self.title_case_headers)
```

Comment:
I'm thinking that perhaps this field shouldn't be logged, since the `State` gets dumped somewhat frequently to help debug the current state, and seeing this longer field being the same constant forever will just be noisy.

Reply:
I agree with that reasoning. Should it be commented like the `method` field to indicate it isn't logged or just get rid of the line entirely?